mmdet/models/dense_heads/gfl_head.py

```diff
@@ -363,6 +365,8 @@ def loss(self,
 
         avg_factor = sum(avg_factor)
         avg_factor = reduce_mean(avg_factor).item()
+        if avg_factor < EPS:
```

Comment:
Shall we use avg_factor.clamp_(min=1) here?

Reply:
Should I change ATSSHead in the same way?

Reply:
You can do that if it is OK with you. Thanks~

Reply:
`avg_factor = reduce_mean(avg_factor).clamp_(min=1).item()`  should work.